fs/cifs/nfs41acl_xdr.h

```diff
@@ -0,0 +1,125 @@
+#ifndef _NFS41ACL_H
+#define _NFS41ACL_H
+
+/*
+ * Native ZFS NFSv41-style ACL is packed in xattr as
+ * follows:
+ *
+ * struct nfsace4i {
+ *    uint32_t type; RFC 5661 Section 6.2.1.1
+ *    uint32_t flag; RFC 5661 Section 6.2.1.4
+ *    uint32_t iflag;
+ *    uint32_t access_mask; RFC 5661 Section 6.2.1.3
+ *    uint32_t who_id;
+ * };
+ *
+ * struct nfsacl4 {
+ *     uint32_t acl_flags; RFC 5661 Section 6.4.3.2
+ *     uint32_t ace_count;
+ *     struct nfsace4i aces<>;
+ * };
+ *
+ * iflag and who_id combined are sufficent for NFS server to convert into ACE
+ * who (RFC 5661 Section 6.2.1.5).
+ */
+
+#define NA41_NAME "system.nfs4_acl_xdr"
+#define NA_TYPE_OFFSET 0
+#define NA_FLAG_OFFSET 1
+#define NA_IFLAG_OFFSET 2
+#define NA_ACCESS_MASK_OFFSET 3
+#define NA_WHO_OFFSET 4
+
+#define ACE4_READ_DATA 0x00000001
```

Comment:
Should there be a comment here referring to RFC 5561 [6.2.1.3](https://datatracker.ietf.org/doc/html/rfc5661#section-6.2.1.3).  ACE Access Mask